gateway/gateway.go

```diff
@@ -265,7 +267,7 @@ func (gateway *HandleT) webRequestBatchDBWriter(process int) {
 				misc.IncrementMapByKey(writeKeySuccessStats, jobWriteKeyMap[uuid], 1)
 				misc.IncrementMapByKey(writeKeySuccessEventStats, jobWriteKeyMap[uuid], jobEventCountMap[uuid])
 			}
-			jobIDReqMap[uuid].done <- err
+			jobIDReqMap[uuid].done <- DBPersistFailed
```

Comment:
What if err is not DBPersistFailed?

Reply:
Let's just log the err and not send specific error as part of client response.